modin/experimental/core/execution/native/implementations/hdk_on_native/partitioning/partition_manager.py

```diff
@@ -178,7 +178,13 @@ def _get_unsupported_cols(cls, obj):
 
             try:
                 at = pyarrow.Table.from_pandas(obj, preserve_index=False)
-            except (pyarrow.lib.ArrowTypeError, pyarrow.lib.ArrowInvalid) as err:
+            except (
+                pyarrow.lib.ArrowTypeError,
+                pyarrow.lib.ArrowInvalid,
+                ValueError,
+            ) as err:
+                if (type(err) == ValueError) and obj.columns.is_unique:
```

Comment:
```suggestion
                if (type(err) == ValueError) and not obj.columns.is_unique:
```
Should this be so?

Reply:
No, the ValueError is raised by pyarrow in case of duplicate columns. We catch and handle this error here. If there are no duplicates (is_unique is True), then error is caused by something different and we just rethrow it.

Reply:
I see, thanks!

Reply:
Maybe one should add a comment explaining this?

Reply:
Added the comment.